frontend/src/app/components/sidebar.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import './sidebar.css'
+import Link from 'next/link'
+import Image from 'next/image'
+
+export default function Sidebar() {
+    return(
+        <div className='sidebar'>
+            <div className='logo'>
+                <Image
+                    width={250} 
+                    height={100} 
+                    src="/UNLV.png"
+                    className='logo'
+                />
+            </div>
+            <Link href="https://www.youtube.com/watch?v=dQw4w9WgXcQ" className='link'>Home</Link>
```

Comment:
I can't seem to click on this link. Can someone else check it and tell me where it leads?